pkg/doc/jose/encrypter_test.go

```diff
@@ -0,0 +1,174 @@
+/*
```

Comment:
are there tests against go-jose?

encrypt with this package and decrypt with go-jose.
encrypt with go-jose and decrypt with this package.

Reply:
it will be done with the Decrypt PR

Reply:
although in this PR, I call go Jose's PraseEncrypted in a test to verify the new Encrypter is creating a go Jose compliant message: https://github.com/hyperledger/aries-framework-go/pull/1683/files#diff-46789f2dc1c2fe2e280ae09f60ac0c82R45